src/patroni.py

```diff
@@ -196,9 +200,11 @@ def get_cluster(
     ) -> dict[str, Any]:
         """Call the cluster endpoint."""
         url = self._get_alternative_patroni_url(attempt, alternative_endpoints)
+        # TODO we don't know the other cluster's ca
+        verify = self._verify if not alternative_endpoints else False
```

Comment:
Same as VM, if we are calling cross cluster in async replication, we cannot know the other cluster's CA.